test/boulder-tools/hotfix_release.sh

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env bash
+
+# -e Stops execution in the instance of a command or pipeline error.
+# -u Treat unset variables as an error and exit immediately.
+set -eu
+
+STATUS="FAILURE"
+
```

Comment:
Instead of manually manipulating the screen, you can use `tput` which will arguably do the same thing, but it's a saner way to interact with terminfo.

```
RESET=$(tput sgr0)
RED=$(tput setaf 1)
GRN=$(tput setaf 2)
```